modules/oxide-components/src/main/ts/components/contexttoolbar/ContextToolbarTypes.tsx

```diff
@@ -0,0 +1,34 @@
+import type { CSSProperties, HTMLAttributes, MouseEventHandler, ReactNode, RefObject } from 'react';
+
+export interface ContextToolbarProps {
+  readonly children: HTMLAttributes<HTMLDivElement>['children'];
+  readonly persistent?: boolean;
+}
+
+export interface ContextToolbarContextValue {
+  readonly isOpen: boolean;
+  readonly open: () => void;
+  readonly close: () => void;
+  readonly triggerRef: RefObject<HTMLDivElement>;
+  readonly toolbarRef: RefObject<HTMLDivElement>;
+  readonly persistent: boolean;
+}
+
+export interface ToolbarProps {
+  readonly children: ReactNode;
+  readonly className?: string;
```

Comment:
I don't think we should expose className or style outside the component. I think all the components we have in `oxide-components` should be styled with `oxide` then any alterations in say size or what ever should be properties similar to `bridge` internally inside the `oxide-components` we can have class and style props but not the library API surface.